niworkflows/reports/core.py

```diff
@@ -137,9 +137,8 @@ def __init__(self, layout, out_dir, config=None):
         if not config:
             raise RuntimeError("Reportlet must have a config object")
 
-        # PY35: Sorted config dict for consistent behavior
         self.name = config.get(
-            "name", "_".join("%s-%s" % i for i in sorted(config["bids"].items()))
+            "name", "_".join("%s-%s" % i for i in config["bids"].items())
```

Comment:
Not sure dropping the `sorted` here is a good move. Given the size of the dictionary to be sorted, I would probably leave it (just remove the comment above) just not to break compatibility with previous runs. WDTY?

Reply:
This is targeting a new minor, so it's fine to break compatibility no? I don't feel strongly about this though, so I can revert.